<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInWidget</class>
 <widget class="QWidget" name="LogInWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>管理员登录</string>
  </property>
  <widget class="QPushButton" name="FaceLoginButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>刷脸登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>账户：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>密码：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="VerificationCodeLabel" name="CodeLable">
          <property name="text">
           <string>验证码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="CodeEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>51</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RepaintButton">
          <property name="minimumSize">
           <size>
            <width>71</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>71</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>刷新验证码</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="logInButton">
        <property name="text">
         <string>登录</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerificationCodeLabel</class>
   <extends>QLabel</extends>
   <header location="global">verificationCodeLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
